corelib-storage/src/main/java/eu/europeana/corelib/solr/derived/AttributionConverter.java

```diff
@@ -292,21 +294,66 @@ private Map concatLangawareMap(Map<String, List<String>> bulkyMap) {
         return flatMap;
     }
 
-    // saves the creator values in a language aware map
-    private Map createCreatorMap(Map<String, List<String>> bulkyMap) {
+    // saves the creator and Title values in a language aware map
+    protected Map createCreatorTitleMap(Map<String, List<String>> bulkyMap) {
         Map<String, List<String>> creatorMap = new HashMap<>();
         if (bulkyMap.get(AttributionConstants.DEF) != null) {
             List<String> defList = stripEmptyStrings(bulkyMap.get(AttributionConstants.DEF));
+            ComparatorUtils.removeDuplicates(defList);
             creatorMap.put("", defList);
+        }
+        if (bulkyMap.get(AttributionConstants.EN) != null) {
+            List<String> enList = stripEmptyStrings(bulkyMap.get(AttributionConstants.EN));
+            getFinalMap(creatorMap, enList, AttributionConstants.EN);
         } else {
-            for (Map.Entry<String, List<String>> langMap : bulkyMap.entrySet()) {
+                getOtherLanguageMap(bulkyMap, creatorMap);
+         }
+        return creatorMap;
+    }
+
+    // checks for languages staring with "en", if not present look for other language.
+    private Map getOtherLanguageMap (Map<String, List<String>> bulkyMap,  Map<String, List<String>> creatorMap) {
+        boolean langValues = false;
+        //check for any language which starts with "en" like "en-GB"
+        for (Map.Entry<String, List<String>> langMap : bulkyMap.entrySet()) {
+            if (! StringUtils.equals(langMap.getKey(), AttributionConstants.DEF) && ! StringUtils.equals(langMap.getKey(), AttributionConstants.EN)
```

Comment:
Why are you checking that if the value not equals to DEF and not equals to EN? 
Basically `if (StringUtils.startsWith(langmap.getKey(), AttributionConstants.EN)` should do it because DEF does not start with "en" and you ruled out that "en" as a value is in the map earlier.